packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 46.14.0 - September 12, 2019
+
+## Fixed
```

Comment:
We should also add an entry for this doc PR that's about to be merged https://github.com/forcedotcom/salesforcedx-vscode/pull/1642